source/js/buyers-guide/components/product-quiz/product-quiz.jsx

```diff
@@ -304,7 +304,13 @@ class ProductQuiz extends Component {
       <div className="tw-flex tw-flex-col medium:tw-flex-col-reverse tw-dark">
         <div className="tw-text-center">
           <div className="tw-mx-auto tw-w-[80px] tw-h-[80px] medium:tw-w-[180px] medium:tw-h-[180px]">
-            <img src={resultType.imgSrc} alt="" width="100%" height="100%" />
+            <img
+              src={resultType.imgSrc}
+              alt=""
+              width="100%"
+              height="100%"
+              loading="eager"
```

Comment:
> Improve quiz result images load speed to be instant the first time (they have a bit of a delay on first load but once they have appeared they load instantly)

Adding `loading="eager"` here which may or may not help with the above task because the image is rendered dynamically by a state update anyway. But it doesn't harm leaving this attribute here...

Reply:
Interesting! I never knew about this attribute 📝